### PR TITLE
Create sparqler.service

### PR DESCRIPTION
**Pull request by Claudenw**
Pull request Description:

Adds sparqler.service as added to sparql.org website.

----

 - [ ] Tests are included.
 - [ ] Documentation change and updates are provided for the [Apache Jena website](https://github.com/apache/jena-site/)
 - [ ] Commits have been squashed to remove intermediate development commit messages.
 - [ ] Key commit messages start with the issue number (GH-xxxx or JENA-xxxx)

By submitting this pull request, I acknowledge that I am making a contribution to the Apache Software Foundation under the terms and conditions of the [Contributor's Agreement](https://www.apache.org/licenses/contributor-agreements.html).

----

See the [Apache Jena "Contributing" guide](https://github.com/apache/jena/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Review by kinow — APPROVED** [review]
Looks good to me!

File: jena-fuseki2/jena-fuseki-main/sparqler/sparqler.service [hunk 1]
```diff
@@ -0,0 +1,39 @@
+## Licensed under the terms of http://www.apache.org/licenses/LICENSE-2.0
+
+#
+# This script runs sparqlr under systemd.
```
Comment (afs) [inline jena-fuseki2/jena-fuseki-main/sparqler/sparqler.service:4]:
```suggestion
# This script runs sparqler under systemd.  
```

File: jena-fuseki2/jena-fuseki-main/sparqler/sparqler.service [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+#  execute "sudo systemctl start sparqler.service"
+# 
+#  Fuseki will now be running and if apache2 is configured properly requests should be redirected to it.
+#
+# Update
+#  execute "sudo systemctl daemon-reload"
+#  execute "sudo systemctl restart sparqler.service
+
+[Unit]
+Description=SPARQLR server
```
Comment (afs) [inline jena-fuseki2/jena-fuseki-main/sparqler/sparqler.service:24]:
```suggestion
Description=SPARQLER server
```

**Comment by afs** [comment]
Is branch `Claudenw-patch-1` inactive?

**Review by afs — APPROVED** [review]

**Comment by afs** [comment]
@Claudenw - can this be merged?

**Comment by afs** [comment]
Ping @Claudenw

**Comment by Claudenw** [comment]
I believe so

**Pull request merged**